etc/di.xml

```diff
@@ -7,5 +7,14 @@
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
-    <preference for="Magento\Framework\App\FrontControllerInterface" type="Magento\UpwardConnector\Controller\Upward" />
+    <type name="Magento\Framework\App\AreaList">
+        <arguments>
+            <argument name="areas" xsi:type="array">
+                <item name="pwa" xsi:type="array">
+                    <item name="frontName" xsi:type="string">pwa</item>
+                </item>
+            </argument>
+        </arguments>
+        <plugin disabled="false" name="Magento_UpwardConnector_Plugin_Magento_Framework_App_AreaList" sortOrder="10" type="Magento\UpwardConnector\Plugin\Magento\Framework\App\AreaList"/>
```

Comment:
Is something else disabling your plugin? Plugins are enabled by default.

Also the name of the plugin is just the class in a different format, I'd follow core and name this something like `upward_connector_app_arealist`

Reply:
Fixed by d8bde4b